### ISSUE TITLE
It's better to remove "artisan:cache:clear" from the default config

### ISSUE THREAD

**Issue by javalaw** [lorisleiva/laravel-deployer#179]
There are some reasons to remove the "artisan:cache:clear" task from the default deployment config.

1. There's no need to clean the cache. In fact, we rarely need to clear the cache when a new version is released. Even if sometimes we need to clear the cache, we just want some specified cache instead of all.

2. it's a VERY DANGEROUS operation. For example, If you use redis as your cache driver, the "artisan cache:clear" command will run "flushdb" command in your redis database. If you share the redis with some other projects, this command will trouble you. And if you use atomic lock with redis, this command will flush them and this will cause some trouble on you business.

So in summary, I think even it's easy that we can remove the commad from the config file manually, it's better to remove the command by default to aviod misunderstand and trouble.

**Comment by lorisleiva** [comment]
Hi there 👋 

I agree! Would you mind making a small PR for this?

**Comment by lorisleiva** [comment]
This has been fixed. I’ll make sure it’s released soon.
